sdk/python/feast/data_source.py

```diff
@@ -384,14 +396,15 @@ def __eq__(self, other):
     def from_proto(data_source: DataSourceProto):
         return KafkaSource(
             name=data_source.name,
+            event_timestamp_column=data_source.timestamp_field,
             field_mapping=dict(data_source.field_mapping),
             bootstrap_servers=data_source.kafka_options.bootstrap_servers,
             message_format=StreamFormat.from_proto(
                 data_source.kafka_options.message_format
             ),
             topic=data_source.kafka_options.topic,
-            event_timestamp_column=data_source.event_timestamp_column,
-            created_timestamp_column=data_source.created_timestamp_column,
+            created_timestamp_column=data_source.timestamp_field,
```

Comment:
is this right? Shouldn't this be `data_source.created_timestamp_column`?

Reply:
Ah good catch, fixed.